contracts/SpeedMarkets/SpeedMarketsAMMData.sol

```diff
@@ -161,7 +161,9 @@ contract SpeedMarketsAMMData is Initializable, ProxyOwned, ProxyPausable {
             for (uint j = 0; j < market.numOfDirections(); j++) {
                 marketDirections[j] = market.directions(j);
                 if (j < market.numOfPrices()) {
-                    marketStrikePrices[j] = market.strikePrices(j);
+                    try market.strikePrices(j) {
+                        marketStrikePrices[j] = market.strikePrices(j);
+                    } catch (bytes memory) {} // fix for old Chained markets where length of prices array is invalid
```

Comment:
empty catch?

Reply:
Here I wanted just to skip fetching strike price from array which doesn't exist. At the beginning of release of Chained markets there was a bug on Market where length of strike prices from market itself was eg. 6 but strikePrices real length is less than 6. This was fixed in the past with new Mastercopy, however fetching data from couple of old chained markets is failing. Here just preventing failure for history data on 2 or 3 chained markets.